LICENSE

```diff
@@ -0,0 +1 @@
+Copyright (c) 2020 FounderTherapy
```

Comment:
https://help.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository

Use the MIT License

Reply:
Updated